.github/workflows/build.yml

```diff
@@ -34,12 +37,17 @@ jobs:
       TEST_TIMEOUT: 900 # 15 minute timeout for tests.
 
     steps:
-    - uses: actions/checkout@v5
+    - name: Harden Runner
```

Comment:
Line 27 says runs-on windows-2022.
Per https://github.com/step-security/harden-runner/blob/main/docs/limitations.md#github-hosted-runners
> Only Ubuntu VM is supported. Windows and MacOS GitHub-hosted runners are not supported. There is a discussion about that [here](https://github.com/step-security/harden-runner/discussions/121).

Hence putting in this step will have no effect other than printing warnings in the job output.